src/main/java/org/springframework/cloud/deployer/spi/mesos/marathon/MarathonAppDeployerProperties.java

```diff
@@ -52,6 +52,11 @@
 	 */
 	private String[] environmentVariables = new String[]{};
 
+	/**
+	 * URIs to set for any deployed app container.
+	 */
+	private String[] uris = new String[]{};
```

Comment:
This can be set as a `List<String>` here and directly copied in the deployer at line 127